package.json

```diff
@@ -10,8 +10,9 @@
   "scripts": {
     "build:demo": "next build demo",
     "cy:open": "cypress open --config-file cypress/config/all.json",
-    "cy:run": "cypress run --config-file cypress/config/all.json",
+    "cy:run": "cypress run --config-file ../cypress/config/ci.json",
```

Comment:
Having this expect to be run from in the demo directory is quite confusing. Is this required?

Reply:
It is required in order to have the paths point to the correct config files in the ci, but I can change the command to be named `ci:cypress` instead so no one tries to run it locally.

Reply:
Aaah, ok. This being because demo doesn;t have a package.json of its own